### DISCUSSION TITLE
权限管理

### DISCUSSION THREAD

**Discussion by yangyingchun1994** [milvus-io/milvus#30893] [Announcement]
如何设置collection级别的权限管理 比如针对某个用户哪个collection 可操作，哪些不可操作

**Comment by yhmo** [comment]
步骤:
https://milvus.io/docs/rbac.md

可设置的权限:
https://milvus.io/docs/users_and_roles.md

Example:
https://github.com/milvus-io/pymilvus/blob/master/examples/role_and_privilege.py
